WebPhapp/backend/server.js

```diff
@@ -24,47 +49,125 @@ app.get('/api/v1/list', (req,res) => {
     console.log('Sent list of items');
 });
 
-// An api endpoint that returns all of the prescriptions
-// associated with a patient ID
-// example: http://localhost:5000/api/v1/prescriptions/01
+/*
+An api endpoint that returns all of the prescriptions associated with a patient ID
+Examples:
+    Directly in terminal:
+        >>> curl "http://localhost:5000/api/v1/prescriptions/01"
+    To be used in Axois call:
+        .get("api/v1/prescriptions/01")
+Returns:
+    A list of prescription objects each with fields: [
+        prescriptionID, patientID, drugID, fillDates, 
+        writtenDate, quantity, daysFor, refillsLeft, 
+        prescriberID, dispenserID, cancelled, cancelDate, drugName
+    ]
+*/
 app.get('/api/v1/prescriptions/:patientID', (req,res) => {
     var patientID = req.params.patientID;
     var prescriptions = readJsonFileSync(
         __dirname + '/' + "dummy_data/prescriptions.json").prescriptions;
 
     var toSend = [];
-    prescriptions.forEach(prescription => {
-        if ( prescription.patientID === patientID ){
-            toSend.push(prescription);
-        }
+    prescriptions.forEach(p => {
```

Comment:
I do not like when appreviations come into play; it makes the code much more difficult to read. Could you change this back to prescription?